forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -21,15 +21,22 @@ import com.joinforage.forage.android.ui.ForageConfig
 import java.util.UUID
 
 /**
- * A class implementation of ForageSDKInterface
+ * An instance of the ForageSDK
```

Comment:
`instance` may not be the most accurate term here.

```suggestion
 * Entry-point to the Forage SDK. A [ForageSDK] instance interacts with the Forage APIs.
 * You need an instance of the [ForageSDK] to perform EBT checkout operations like:
 * - ... LIST and link tokenizeEBTCard, checkBalance, etc. in KDoc style
 ```